File: src/ssr_py_pkg/scripts/yao_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String


if __name__== "__main__":
    rospy.init_node("yao_node")
    rospy.logwarn("大厨TM我来了!")

    pub =rospy.Publisher("gie_gie_kuai_dai_wo",String,queue_size=10)

    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        rospy.loginfo("我要开始刷屏了")
        msg=String()
        msg.data="求带飞+V"
        pub.publish(msg)
        rate.sleep() 

File: src/ssr_pkg/src/yao_node.cpp
#include <ros/ros.h>
#include <std_msgs/String.h>

int main(int argc, char *argv[])
{
    ros::init(argc,argv,"yao_node");
    printf("顶级厨师我TM来了!\n");

    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("gie_gie_kuai_dai_wo",10);

    ros::Rate loop_rate(10);

    while(ros::ok())
    {
        printf("我要刷屏了！\n");
        std_msgs::String msgs;
        msgs.data = "求上分+V";
        pub.publish(msgs);
        loop_rate.sleep();
    } 
    return 0;
}
 
 
 